### ISSUE TITLE
Misleading errors when [linter]_dir doesn't exist

### ISSUE THREAD

**Issue by javajawa** [wearerequired/lint-action#142]
In [a new project](https://github.com/javajawa/py-tiny-orm/blob/c0d72e8/.github/workflows/lint.yml) I tried to use the lint action against multiple folders, be providing a space separate string to the `[linter]_dir` parameter (thinking they were passed through directly to the program.

For Flake8, this caused the tool to be reported as not installed:

```
2021-01-01T00:39:05.7435588Z Installing collected packages: zipp, typed-ast, toml, regex, pyflakes, pycodestyle, pathspec, mccabe, importlib-metadata, click, appdirs, mypy, flake8, black
2021-01-01T00:39:08.1364102Z Successfully installed appdirs-1.4.4 black-20.8b1 click-7.1.2 flake8-3.8.4 importlib-metadata-3.3.0 mccabe-0.6.1 mypy-0.790 pathspec-0.8.1 pycodestyle-2.6.0 pyflakes-2.2.0 regex-2020.11.13 toml-0.10.2 typed-ast-1.4.2 zipp-3.4.0
2021-01-01T00:39:08.1465970Z WARNING: You are using pip version 20.3.1; however, version 20.3.3 is available.
2021-01-01T00:39:08.1468713Z You should consider upgrading via the '/opt/hostedtoolcache/Python/3.7.9/x64/bin/python -m pip install --upgrade pip' command.

2021-01-01T00:39:08.2459139Z ##[group]Run which flake8
2021-01-01T00:39:08.2459835Z which flake8
2021-01-01T00:39:08.2511901Z shell: /bin/bash -e {0}
2021-01-01T00:39:08.2512430Z env:
2021-01-01T00:39:08.2513922Z   pythonLocation: /opt/hostedtoolcache/Python/3.7.9/x64
2021-01-01T00:39:08.2514630Z ##[endgroup]

2021-01-01T00:39:08.2758912Z /opt/hostedtoolcache/Python/3.7.9/x64/bin/flake8

2021-01-01T00:39:08.2986243Z ##[group]Run samuelmeuli/lint-action@v1
2021-01-01T00:39:08.2986784Z with:
[...]
2021-01-01T00:39:08.3012884Z ##[endgroup]
2021-01-01T00:39:08.3631035Z SHA of last commit is "c0d72e8fe5dde1167d95564c5991f07400581086"
2021-01-01T00:39:08.3632162Z ##[group]Run Flake8
2021-01-01T00:39:08.3638942Z Verifying setup for Flake8…
2021-01-01T00:39:08.3741885Z ##[error]Error: Flake8 is not installed
2021-01-01T00:39:08.3757445Z /home/runner/work/_actions/samuelmeuli/lint-action/v1/dist/index.js:1108
2021-01-01T00:39:08.3761344Z 	throw new Error(`Exiting because of unhandled promise rejection`);
```

For MyPy, a `trim()` call was called against a null

```
2021-01-01T00:17:25.5524734Z Verifying setup for Mypy…
2021-01-01T00:17:25.5637835Z Verified Mypy setup
2021-01-01T00:17:25.5639033Z Will use Mypy to check the files with extensions py
2021-01-01T00:17:25.5640671Z Linting files in /home/runner/work/py-tiny-orm/py-tiny-orm/orm examples with Mypy…
2021-01-01T00:17:25.5671668Z ##[error]TypeError: Cannot read property 'trim' of null
2021-01-01T00:17:25.5684623Z /home/runner/work/_actions/samuelmeuli/lint-action/v1/dist/index.js:1108
2021-01-01T00:17:25.5687407Z 	throw new Error(`Exiting because of unhandled promise rejection`);
2021-01-01T00:17:25.5687969Z 	^
2021-01-01T00:17:25.5688199Z 
```

**Comment by ocean90** [comment]
Hi, thanks for the report. The [linter]_dir option should only contain a single path which is used as the working directory for where the linter command should be run.

**Comment by javajawa** [comment]
Indeed. The issue is not a request to change that behaviour, but about the error messages being returns when the directory doesn't exist; the error I would expect in this case is `directory 'src test' does not exist`, not `flake8 is not installed`.
